hudi-client/hudi-spark-client/src/main/java/org/apache/hudi/table/action/commit/SparkInsertOverwriteCommitActionExecutor.java

```diff
@@ -66,11 +65,6 @@ protected Partitioner getPartitioner(WorkloadProfile profile) {
     return new SparkInsertOverwritePartitioner(profile, context, table, config);
   }
 
-  @Override
-  protected String getCommitActionType() {
```

Comment:
@satishkotha greetings !

with reference to the function! 
https://github.com/apache/hudi/blob/03668dbaf1a60428d7e0d68c6622605e0809150a/hudi-client/hudi-spark-client/src/main/java/org/apache/hudi/table/action/commit/SparkInsertOverwriteCommitActionExecutor.java#L70

Why specifically we need `HoodieTimeline.REPLACE_COMMIT_ACTION` for `SparkInsertOverwriteCommitActionExecutor` ?

If you can enlighten it would be really helpful